config/toml.go

```diff
@@ -397,6 +397,11 @@ type = "flood"
 # you can disable rechecking.
 recheck = {{ .Mempool.Recheck }}
 
+# RecheckTimeout is the time the application has to return CheckTx responses after the
```

Comment:
Maybe we should write a sentence explaining how an APP developer is supposed to measure / tweak this parameter? Or the default 3s is good for the majority of the existing applications?

Reply:
I added a paragraph to `config.toml.md`.